notify/options.go

```diff
@@ -102,7 +102,7 @@ func WithPeriod(period time.Duration) Option {
 // notably the public and private mail API keys. This is always
 // required, unless testing.
 func WithSecrets(secrets map[string]string) Option {
```

Comment:
This option is the only option that is specific to Mailjet. It could be renamed WithMailjetSecrets.

Reply:
I'm thinking we should wait until we have another implementation; there'll have to be a broader discussion about how we organise/name the options then.

Reply:
Okay. All implementations will require secrets of some kind. There is also something to be said for brevity of names

Reply:
Yes, the brevity is nice; if we want to maintain that when additional implementations appear, we'll need to "scope" the options so that there's no name clashes. This could be done by creating mailjet and sms packages as required so that each can have options of the same name.